sklego/model_selection.py

```diff
@@ -17,36 +17,42 @@ class TimeGapSplit:
     """
     Provides train/test indices to split time series data samples.
     This cross-validation object is a variation of TimeSeriesSplit with the following differences:
+
```

Comment:
Overall I tried to keep the consistent behaviour of not refering to self in the docstring